rocm/build.sh

```diff
@@ -117,6 +117,12 @@ popd
 
 
 # hip
+if (( ROCM_MAJOR_MINOR >= 602 )); then
+  HIP_DIR=hip-${ROCM_VERSION}
+else
+  HIP_DIR=HIP-${ROCM_VERSION}
+fi
+rm -rf clr-${ROCM_VERSION} ${HIP_DIR}
```

Comment:
I'm confused as to why you need to delete things here... but ../ I'll give it a go

Reply:
got it; thanks